plugins/content_distribution/lib/model/EntryDistribution.php

```diff
@@ -70,7 +70,7 @@ public function getNextReport($format = 'Y-m-d H:i:s')
 	/**
 	 * @return entry
 	 */
-	protected function getEntry()
+	public function getEntry()
 	{
 		$entryId = $this->getEntryId();
```

Comment:
This code is useless  , Since retrieveByPKNoFilter already searches in the pool.
Therefor you just call return retrieveByPKNoFilter(entryId)